### PR TITLE
#112 add View interface and implement it in the Console class

### PR DESCRIPTION
**Pull request by kortov**
#112

### CODE REVIEW

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package juja.sqlcmd.view;
+
+import java.util.Scanner;
+
+public class Console implements View {
+    Scanner scanner;
```
Comment (AVM82) [inline src/main/java/juja/sqlcmd/view/Console.java]: Perhaps we can make the Scanner scanner class field private

**Review by AVM82 — COMMENTED** [review]
Not sure, but it seems to me that there is no need to test the Console class at the bottom stage

File: src/main/java/juja/sqlcmd/view/Console.java (hunk 1 above)
Comment (kortov) [reply src/main/java/juja/sqlcmd/view/Console.java]: Yep)

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import java.util.Scanner;
 
 public class Console implements View {
-    Scanner scanner;
+    private Scanner scanner;
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/view/Console.java]: Why to have a Scanner as a field? You don't return it anywhere and will never reuse somewhere else... I think it's redundant

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 3]
```diff
@@ -3,7 +3,7 @@
...
 
     public Console() {
```
Comment (Shtramak) [inline src/main/java/juja/sqlcmd/view/Console.java]: If you will delete a field then empty constructor will be enough. Also pay your attention that you initialize scanner field twice: once in a constructor and then in read() method

File: src/main/java/juja/sqlcmd/view/Console.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    public Console(InputStream inputStream, OutputStream outputStream) {
+        this.scanner = new Scanner(inputStream);
+        this.outputStream = outputStream;
+    }
+
+    @Override
+    public void write(String message) {
+        try {
+            String modifiedToMimicUserInput = message + System.lineSeparator();
+            outputStream.write(modifiedToMimicUserInput.getBytes());
```
Comment (AVM82) [inline src/main/java/juja/sqlcmd/view/Console.java:25]: You can write this in one line

**Review by AVM82 — APPROVED** [review]
good

**Review by adaleorion — APPROVED** [review]

**Pull request merged**